database/getters_test.go

```diff
@@ -63,3 +62,36 @@ func TestGet_DatabaseName(t *testing.T) {
 		})
 	}
 }
+
+func TestGet_TableColumns(t *testing.T) {
+	type fields struct {
+		DB *Database
+	}
+	type args struct {
+		t string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    []string
+		wantErr bool
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			g := &Get{
+				DB: tt.fields.DB,
+			}
+			got, err := g.TableColumns(tt.args.t)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("TableColumns() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !reflect.DeepEqual(got, tt.want) {
```

Comment:
The compiler throws warning here stating not to use DeepEqual to compare errors. I will work on the testing after we merge this.

Reply:
ok sounds good.